webdriver/tests/bidi/browsing_context/locate_nodes/sandbox.py

```diff
@@ -0,0 +1,36 @@
+
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget
+
+
+@pytest.mark.asyncio
+async def test_locate_nodes_in_sandbox(bidi_session, inline, top_context):
+    url = inline("""<div data-class="one">foobarBARbaz</div><div data-class="two">foobarBARbaz</div>""")
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"], url=url, wait="complete"
+    )
+
+    result = await bidi_session.browsing_context.locate_nodes(
+        context=top_context["context"],
+        locator={ "type": "css", "value": "div[data-class='one']" },
+        sandbox="sandbox"
+    )
+
+    assert result["context"] == top_context["context"]
+    assert result["nodes"].length == 1
+    node_id = result["nodes"][0]["sharedId"]
+
+    # Check that changes are not present in sandbox
```

Comment:
I have troubles to understand this particular test. Maybe you can explain it a bit more?

We probably want to also test that retrieving a node from different sandboxes (or the default one) will return a node with the same `sharedId`.

Reply:
This comment was an error. I've removed it. I also added tests for different sandboxes as you suggested.